### PR TITLE
Feature ETP-453: Fix JWT token validation runtime error

### PR DESCRIPTION
**Pull request by LuuchoRocha**
Feature ETP-453: Fix JWT token validation runtime error

### CODE REVIEW

**Review by Gremiger — APPROVED** [review]

File: src/com/etendoerp/metadata/BaseServlet.java [hunk 1]
```diff
@@ -43,12 +43,17 @@ public final void service(HttpServletRequest request, HttpServletResponse respon
...
-        String token = getToken(request);
-        DecodedJWT decodedToken = decodeToken(token);
+    protected DecodedJWT getDecodedToken(HttpServletRequest request) {
+        try {
+            String token = getToken(request);
+            DecodedJWT decodedToken = decodeToken(token);
 
-        if (decodedToken != null) return decodedToken;
-        else throw new UnauthorizedException();
+            if (decodedToken != null) return decodedToken;
```
Comment (Gremiger) [inline src/com/etendoerp/metadata/BaseServlet.java:51]: I prefer to have the curly braces even if it's a one-line if/else expression, but besides that, LGTM

Comment (LuuchoRocha) [reply src/com/etendoerp/metadata/BaseServlet.java:51]: I just wanted to save one line 😅
Jokes aside, you're right. 
I agree, in terms of readability is better to always keep the curly braces.
Thanks for your feedback!

Comment (Matias-Bernal) [reply src/com/etendoerp/metadata/BaseServlet.java:51]: It's due to a Java standard convention: https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#449

Comment (LuuchoRocha) [reply src/com/etendoerp/metadata/BaseServlet.java:51]: Shame on me, I didn't know. 
I'll fix all the bad practices I've applied and update the PR 

Many thanks for your help!

**Comment by Matias-Bernal** [comment]
Mergeaste este PR sin tener los 2 approvals? Ademas no veo configurado la ejecucion de Sonar y el bot de review.

**Pull request merged**